src/components/Loading/FullscreenLoading.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import {StyleSheet, View} from 'react-native';
+import LoadingIndicator from './LoadingIndicator';
+
+const FullScreenLoadingIndicator = () => (
+    <View style={styles.container}>
+        <LoadingIndicator size={88} />
+    </View>
+);
+
+const styles = StyleSheet.create({
```

Comment:
Just a heads up, we have most styles just in a big object for now and not yet using `StyleSheet.create()` - see `styles.js` file.